proto/bragi/cs2.proto

```diff
@@ -42,6 +42,7 @@ message CS2MatchSnapshot {
   DataStatus data_status = 6;
   repeated Announcement announcements = 4;
   CS2MatchState match_state = 5;
+  bragi.Tournament tournament  = 7;
```

Comment:
`Tournament tournament = 7;` should be enough.

Occurs multiple times

Reply:
Fixed